### PR TITLE
Check answers

### PR DESCRIPTION
**Pull request by goodviber**
### Changes proposed in this pull request
Refactoring the check answers pages

### CODE REVIEW

File: app/helpers/session_helper.rb [hunk 1]
```diff
@@ -26,4 +26,15 @@ def show_name_and_email
     "#{session[:registration]['first_name'].capitalize}" + " " + "#{session[:registration]['last_name'].capitalize}" + ", " + "#{session[:registration]['email_address']}"
   end
 
+  def show_name
```
Comment (ethax-ross) [inline app/helpers/session_helper.rb:29]: Can we add tests for these

File: app/models/primary_or_secondary.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class PrimaryOrSecondary < Base
   attribute :primary_or_secondary, :string
 
-  validates :primary_or_secondary, inclusion: { in: %w(primary secondary), message: "Please answer Primary or Secondary." }
+  validates :primary_or_secondary, inclusion: { in: %w(primary secondary), message: "You must select an option" }
```
Comment (ethax-ross) [inline app/models/primary_or_secondary.rb:4]: Might be worth making your own inclusion/presence validators so you don't have to repeat the same message everywhere

File: app/helpers/session_helper.rb (hunk 1 above)
Comment (goodviber) [reply app/helpers/session_helper.rb:29]: Yep

File: app/models/primary_or_secondary.rb (hunk 2 above)
Comment (goodviber) [reply app/models/primary_or_secondary.rb:4]: Good idea

File: .gitignore [hunk 3]
```diff
@@ -25,6 +25,8 @@
 /vendor/cache
 yarn-debug.log*
 .yarn-integrity
+#spec state
+/spec/examples.txt
```
Comment (ethax-ross) [inline .gitignore:29]: I thought this was something to do with test parallelisation in Rails 6 🙄

**Pull request merged**